extensions/visualizations/visualizations.js

```diff
@@ -74,3 +74,28 @@ oppia.directive('oppiaVisualizationFrequencyTable', [function() {
     ]
   };
 }]);
+
+oppia.directive('oppiaVisualizationEnumeratedFrequencyTable', [function() {
+  return {
+    restrict: 'E',
+    scope: {},
+    templateUrl: 'visualizations/EnumeratedFrequencyTable',
+    controller: [
+      '$scope', '$attrs', 'HtmlEscaperService',
+      function($scope, $attrs, HtmlEscaperService) {
+        $scope.data = HtmlEscaperService.escapedJsonToObj($attrs.data).map(
+          function(row, i) {
+            row.displayName = '#' + (i + 1);
+            row.answerId = 'answer-' + i;
+            return row;
+          });
+        $scope.options = HtmlEscaperService.escapedJsonToObj($attrs.options);
+        $scope.toggleRowAnswer = function(answerId) {
+          var answer = document.getElementById(answerId);
```

Comment:
Don't do this, use ng-style instead. Maintain a separate $scope variable here (list of bools, perhaps) to track whether an answer's displayed or not.

In general, don't do document.getElementById() or other DOM stuff within JS functions in Angular.

Reply:
Done.